07_wait_for_cluster.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+source logging.sh
+source common.sh
+
+export KUBECONFIG=ocp/$CLUSTER_NAME/auth/kubeconfig
+
+workers=$NUM_WORKERS
+if $APPLY_EXTRA_WORKERS ; then
```

Comment:
Probably safer to make this an actual test?

Reply:
In disconnected environment, marketplace operators are in degraded state and this will cause infinite loop in local environment. I think, we can add `OPENSHIFT_CI` check in here to skip this step in local. It has already existed in CI.

https://github.com/openshift/release/blob/be015aa60c8b7641d8f71fae6dad64687b1a3645/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L77